src/App.js

```diff
@@ -1,25 +1,65 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import logo from "./logo.svg";
+import Recipe from "./Recipe";
+import "./App.css";
+
+const App = () => {
+  const APP_ID = "3ab50fdd";
+  const APP_KEY = "b1e9471936f7ffbb467f55bd471aef3b";
+
+  const [recipes, setRecipes] = useState([]);
+  const [search, setSearch] = useState("");
+  const [query, setQuery] = useState("chicken");
+
+  useEffect(() => {
+    getRecipes();
+  }, [query]);
+
+  const getRecipes = async () => {
+    const response = await fetch(
+      `https://api.edamam.com/search?q=${query}&app_id=${APP_ID}&app_key=${APP_KEY}`
+    );
+    const data = await response.json();
+
+    setRecipes(data.hits);
+  };
+
+  const updateSearch = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const getSearch = (e) => {
+    e.preventDefault();
+    setQuery(search);
+    setSearch("");
+  };
 
-function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <form onSubmit={getSearch} className="search-form">
+        <input
+          onChange={updateSearch}
+          className="search-bar"
+          type="text"
+          value={search}
+        ></input>
```

Comment:
if there's nothing inside the input tags, you can just use a self closing tag, as in you would do:
<input
          onChange={updateSearch}
          className="search-bar"
          type="text"
          value={search}
        />